test/benchmark.js

```diff
@@ -13,39 +13,32 @@ const testDataWrite = [
 ]
 
 const {firstVersion, lastVersion} = require('./common/parallel')
-console.log({firstVersion, lastVersion})
+console.log(`Testing versions from ${firstVersion} to ${lastVersion}`)
 
 mc.supportedVersions.forEach(function (supportedVersion, i) {
   if (!(i >= firstVersion && i <= lastVersion)) { return }
 
   const mcData = require('minecraft-data')(supportedVersion)
   const version = mcData.version
-  describe('benchmark ' + version.minecraftVersion, function () {
+  describe(`Benchmark for ${version.minecraftVersion}`, function () {
     this.timeout(60 * 1000)
     const inputData = []
-    it('bench serializing', function (done) {
+    it('Bench serializing', function (done) {
       const serializer = mc.createSerializer({state: states.PLAY, isServer: false, version: version.minecraftVersion})
-      let start, i, j
-      console.log('Beginning write test')
```

Comment:
Why did you remove these console.log ?

Reply:
What's the reason to have them?

Reply:
To know how much time the benchmark take ? It's a benchmark !

Reply:
Mocha already tells you